### ISSUE TITLE
TypeError: Cannot read property '$nin' of undefined

### ISSUE THREAD

**Issue by dfischer** [feathersjs/feathers-sequelize#45]
Example query:

```
const players = await this.app.service('players').find({query: {
  assigneeUserId: null,
}});
```

```
TypeError: Cannot read property '$nin' of null 
lib/utils.js:67:14
```

https://github.com/feathersjs/feathers-sequelize/blob/master/src/utils.js#L49 no null checking here 

Intention is to get a null value out of sequelize / row / value.

**Comment by againer** [comment]
+1

Running into the same issue.  Need to be able to query on null fields.

**Comment by daffl** [comment]
Fixed in v1.1.6
